lib/package-manager/get-binary.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const path = require('path');
+const which = require('which').sync;
+const npmWhich = require('npm-which')(__dirname).sync;
+
+const windows = (process.platform === 'win32');
+
+module.exports = function(binaryName, options) {
```

Comment:
nit: give the function a name? `getBinary`?

Reply:
good call, done